example/nsfw_demo/src/App.js

```diff
@@ -138,16 +138,22 @@ class App extends Component {
   }
 
   detectWebcam = async () => {
-    const video = document.querySelectorAll('.captureCam')
-    const predictions = await this.state.model.classify(video[0])
-    let droppedImageStyle = this.detectBlurStatus(predictions[0].className)
-    this.setState({
-      message: `Identified as ${predictions[0].className}`,
-      predictions,
-      droppedImageStyle
-    })
+      // await this.sleep(100)
```

Comment:
@GantMan here is a strange behaviour.

If I dont add sleep 100,
The predictions are not updated.

If I add sleep 100, predictions are updated, But, they update for sometime even after I close the webcam.

<img width="624" alt="screenshot 2019-03-04 22 43 49" src="https://user-images.githubusercontent.com/3761745/53750709-8959e380-3ed0-11e9-8f12-10e6d1f51566.png">

Reply:
no update if not made sleep 100

<img width="397" alt="screenshot 2019-03-04 22 43 06" src="https://user-images.githubusercontent.com/3761745/53750766-a7274880-3ed0-11e9-9787-27073ecdf3ee.png">

Reply:
predictions aren't updated at all if I don't set it to sleep for 100ms.